Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -23,6 +24,18 @@ namespace aod
 // the o2::aod namespace.
 DECLARE_SOA_STORE();
 
+// This is required to make
+//  o2::aod::datamodel::getTreeName
+//  o2::aod::datamodel::getColumnNames
+// available
+// ATTENTION: getTreeName and getColumnNames must be updated
+//            in accordance with changes of the AnalysisDataModel!
+namespace datamodel
+{
+std::string getTreeName(header::DataHeader dh);
+std::vector<std::string> getColumnNames(header::DataHeader dh);
+} // namespace datamodel
+
```

Comment:
Please keep this private API, at least for now.

Reply:
getTreeName is used in DataInputDirector.cxx and getColumnNames in AODReaderHelpers.cxx. I therefore declare these functions in DataAnalysisModel.h and include the header file in the DataInputDirector.cxx and AODReaderHelpers.cxx. How else would I do this?

Reply:
Put it in a header file in `src/` and include that header file in the needed .cxx. No need to pollute the `AnalysisDataModel.h` (which every single analysis will include) with helpers which are relevant only for the implementation, in particular since they require `Headers/DataHeader.h` which is extremely heavy on the compiler.

Reply:
is it ok to use src/AnalysisDataModel.h or better to rename everything to src/AnalysisDataModelHelpers.[cxx,h]?

Reply:
Add "Helpers", please.

Reply:
done.